ravenframework/OutStreams/PlotInterfaces/OptPath.py

```diff
@@ -103,6 +104,7 @@ def run(self):
       @ In, None
       @ Out, None
     """
+    self.counter += 1
```

Comment:
this needs to be implemented for each specialized plotter we make, right? Is there a general way we can handle this so that it's not a hidden option for new plot development?

Reply:
looks great, thank you!